### DISCUSSION TITLE
Code Server terminal shows black screen

### DISCUSSION THREAD

**Discussion by HemantGujarati** [coder/code-server#7108] [Q&A]
Hey, 
The terminal shows black screen everytime I open terminal . Can anyone help me.
![Screenshot_20241209-193029](https://github.com/user-attachments/assets/ec221d90-ccbb-43bb-9149-ed2fcfe3cd31)

**Comment by code-asher** [comment]
Are there any errors in the browser console or in code-server's output?

**Reply by HemantGujarati** [reply]
I'm getting these warnings, but I don't think these have anything to with the terminal.
The code is running without any problem, it's just the terminal is showing black screen.![Screenshot_20241211-130101.png](https://github.com/user-attachments/assets/8d71ecb4-eb91-4b1c-adc9-8c07423b4dc8)

**Reply by HemantGujarati** [reply]
> I'm getting these warnings, but I don't think these have anything to with the terminal.
> The code is running without any problem, it's just the terminal is showing black screen.![Screenshot_20241211-130101.png](https://github.com/user-attachments/assets/8d71ecb4-eb91-4b1c-adc9-8c07423b4dc8)
> 
> 

I solved this by changing the value to 'none' instead of 'jedi' or 'pylance'. Is that okay to do?

**Reply by code-asher** [reply]
I think that will just make it so you will not get any language server running, so things like refactoring or jumping to definition will not work.

You will probably want to check both the "window" and "extension host" logs (click the "go to output" button).  There may be information about the terminal in those logs as well.

**Reply by HemantGujarati** [reply]
@code-asher thanks for your help in solving the problem, I really appreciate it! 
The problem is solved after i reinstalled the whole thing and also reset chrome to default.
I think the problem was with chrome, using dark theme.
